wallet/lib/pages/purchase_item/widgets/accept_policy_bottom_sheet.dart

```diff
@@ -0,0 +1,195 @@
+                  alignment: Alignment.bottomCenter,
+                  child: ClipPath(
+                    clipper: LeftRightTopClipper(),
+                    child: Container(
+                      height: 0.33.sh,
+                      width: double.infinity,
+                      color: AppColors.kMainBG,
+                      child: Column(
+                        children: [
+                          SizedBox(
+                            height: 60.h,
+                            child: Image.asset(
+                              ImageUtil.PYLONS_LOGO,
+                            ),
+                          ),
+                          MyCheckBox(
+                            LocaleKeys.acknowledge_terms_service.tr(),
+                            isSelected: viewModel.isCheckTermServices,
+                            onChange: (value) {
+                              viewModel.toggleCheckTermServices(value!);
+                            },
+                            onLinkTap: () {
+                              /// TO DO
+                              /// Don't have url of term & services
+                            },
+                          ),
+                          SizedBox(
+                            height: 15.0.h,
+                          ),
+                          MyCheckBox(
+                            LocaleKeys.acknowledge_privacy_policy.tr(),
+                            isSelected: viewModel.isCheckPrivacyPolicy,
+                            onChange: (value) {
+                              viewModel.toggleCheckPrivacyPolicy(value!);
+                            },
+                            onLinkTap: () => launchUrlString(kPrivacyPolicyLink),
+                          ),
+                          SizedBox(
+                            height: 25.0.h,
+                          ),
+                          PylonsGetStartedButton(
+                            key: const Key(kAcceptBottomSheetBtnKey),
+                            enabled: viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy,
+                            onTap: onGetStarted,
+                            text: LocaleKeys.get_started.tr(),
+                            loader: ValueNotifier(false),
+                            fontWeight: FontWeight.normal,
+                            btnHeight: 40,
+                            btnWidth: 260,
+                            btnUnselectBGColor: AppColors.kDarkDividerColor,
+                            fontSize: 14,
+                            textColor: !(viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy) ? AppColors.kUserInputTextColor : AppColors.kWhite,
+                          ),
+                        ],
+                      ),
+                    ),
+                  ),
+                );
+              },
+            );
+          },
+        )
+      ],
+    );
+  }
+}
+
+class MyCheckBox extends StatelessWidget {
```

Comment:
Why dont we use existing checkbox?

Reply:
Checkbox design is change so for that i have to make a new checkbox to implement that design